src/components/CommentList/CommentList.jsx

```diff
@@ -2,0 +1,17 @@
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) =>
+  comments.length === 0 ? (
+    <>
+      <hr />
+
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    </>
+  ) : (
+    <div className="CommentList">
+      {comments.map(comment => (
+        <CommentInfo comment={comment} key={comment.id} />
```

Comment:
The `key` prop is correctly set to `comment.id`. Ensure that each comment object has a unique `id` to prevent rendering issues.